src/patternfly/components/FormControl/form-control.scss

```
@@ -263,196 +187,177 @@ $pf-v5-c-form-control--m-clock--Coordinates: "M256 8C119 8 8 119 8 256s111 248 2
 
   &:focus {
     --#{$form-control}--BorderBottomColor: var(--#{$form-control}--focus--BorderBottomColor);
+    --#{$form-control}__icon--Height: calc(var(--#{$form-control}--Height) - var(--#{$form-control}--focus--BorderBottomWidth));
+    --#{$form-control}__toggle-icon--Height: calc(var(--#{$form-control}--Height) - var(--#{$form-control}--focus--BorderBottomWidth));
+    --#{$form-control}--PaddingBottom: var(--#{$form-control}--focus--PaddingBottom);
 
-    padding-bottom: var(--#{$form-control}--focus--PaddingBottom); // Can't redefine --#{$form-control}--PaddingBottom b/c the hover padding bottom uses the focus padding var to compute it's padding
     border-bottom-width: var(--#{$form-control}--focus--BorderBottomWidth);
   }
 
   &.pf-m-expanded {
     --#{$form-control}--BorderBottomColor: var(--#{$form-control}--m-expanded--BorderBottomColor);
+    --#{$form-control}__icon--Height: calc(var(--#{$form-control}--Height) - var(--#{$form-control}--m-expanded--BorderBottomWidth));
+    --#{$form-control}__toggle-icon--Height: calc(var(--#{$form-control}--Height) - var(--#{$form-control}--m-expanded--BorderBottomWidth));
+    --#{$form-control}--PaddingBottom: var(--#{$form-control}--m-expanded--PaddingBottom);
 
-    padding-bottom: var(--#{$form-control}--m-expanded--PaddingBottom); // Can't redefine --#{$form-control}--PaddingBottom b/c the hover padding bottom uses the focus padding var to compute it's padding
     border-bottom-width: var(--#{$form-control}--m-expanded--BorderBottomWidth);
   }
 
-  &:disabled {
+  &.pf-m-disabled {
     --#{$form-control}--BackgroundColor: var(--#{$form-control}--disabled--BackgroundColor);
+    --#{$form-control}--Color: var(--#{$form-control}--disabled--Color);
+    --#{$form-control}__toggle-icon--Color: var(--#{$form-control}--disabled__toggle-icon--Color);
```

Just tagged a few here, but I think there are others that should be updated to `--m-disabled`

```suggestion
  &.pf-m-disabled {
    --#{$form-control}--BackgroundColor: var(--#{$form-control}--m-disabled--BackgroundColor);
    --#{$form-control}--Color: var(--#{$form-control}--m-disabled--Color);
    --#{$form-control}__toggle-icon--Color: var(--#{$form-control}--m-disabled__toggle-icon--Color);
```